clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -690,6 +690,81 @@ struct Table {
   }
 };
 
+// Add any llvm-spirv option that rely on a specific Triple in addition
+// to user supplied options.
+// NOTE: Any changes made here should be reflected in the similarly named
+// function in clang/lib/Driver/ToolChains/Clang.cpp.
+static void
+getTripleBasedSPIRVTransOpts(const ArgList &Args,
+                             SmallVector<StringRef, 8> &TranslatorArgs,
+                             const llvm::Triple Triple) {
+  bool IsCPU = Triple.isSPIR() &&
+               Triple.getSubArch() == llvm::Triple::SPIRSubArch_x86_64;
+  // Enable NonSemanticShaderDebugInfo.200 for CPU AOT and for non-Windows
```

Comment:
Double comment, we probably only want one right?

Reply:
Not something I introduced. But fixed it. Thanks